mobile/src/components/MobileAppRoute.jsx

```diff
@@ -16,7 +18,7 @@ const MobileAppRoute = requireSetup => (
         <Route path='settings' name='mobile.settings' component={Settings} />}
       </Route>
     </Route>
-    <Route path='onboarding' component={OnBoarding} />
+    <Route path='onboarding' component={OnBoarding} onLeave={() => initBar()} />
```

Comment:
@kosssi or @goldoraf I am not sure it is a good way to init the bar after the onboarding steps. What do you think?